Adding instructions for configuring Facebook Groups app to marketplace page. (#6971)
PipedreamHQ/pipedream: components/facebook_groups/README.md

# Getting Started

> In order to connect your Facebook Group to Pipedream, you'll need to navigate to the group which you are an administrator of, and in the lefthand sidebar, under **Settings** > **Group Settings**, you will find **Manage Advanced Settings**. Click the edit button under **Apps**.

Reviewer: ```suggestion
In order to connect your Facebook Group to Pipedream, you need to navigate to the group for which you are an administrator, and in the left sidebar, under **Settings** > **Group Settings**, you will find **Manage Advanced Settings**. Click the Edit button under **Apps**.
```

Reviewer: Changed 'you'll need to' to 'you need to' (Google's Developer docs style guide: contractions), Changed 'lefthand' to 'left' (Dreyer's English: simplicity)

> You need to install the Pipedream app in your Facebook Group to use any prebuilt triggers or actions. To install Pipedream, navigate to the group where you're an administrator, and in the left sidebar, under **Settings** > **Group Settings**, you'll find **Manage Advanced Settings**. Click the Edit button under **Apps**.

Reviewer 2: >to use any prebuilt triggers or actions

I thought it was, "to make API requests to Facebook Groups and use any pre-built trigger or actions"

> You need to install the Pipedream app in your Facebook Group in order to make any API calls, or use any prebuilt triggers or actions. To install Pipedream, navigate to the group for which you are an administrator, and in the left sidebar, under **Settings** > **Group Settings**, you will find **Manage Advanced Settings**. Click the Edit button under **Apps**.

Reviewer: ```suggestion
You need to install the Pipedream app in your Facebook Group to use any prebuilt triggers or actions. To install Pipedream, navigate to the group where you're an administrator, and in the left sidebar, under **Settings** > **Group Settings**, you'll find **Manage Advanced Settings**. Click the Edit button under **Apps**.
```

Reviewer: Changed 'will need' to 'need', 'you are' to 'you're', 'will find' to 'you'll find' - Google's Developer docs style guide (contractions, future tense)

> You need to install the Pipedream app in your Facebook Group in order to make any API calls, or use any prebuilt triggers or actions. To install Pipedream, navigate to the group for which you are an administrator, and in the left sidebar, under **Settings** > **Group Settings**, you will find **Manage Advanced Settings**. Click the Edit button under **Apps**.

Reviewer: ```suggestion
You need to install the Pipedream app in your Facebook Group to make any API calls, or use any prebuilt triggers or actions. To install Pipedream, navigate to the group where you're an administrator, and in the left sidebar, under **Settings** > **Group Settings**, you'll find **Manage Advanced Settings**. Click the Edit button under **Apps**.
```

Reviewer: Changed 'in order to' to 'to', 'you are' to 'you're', 'will find' to 'you'll find' - Google's Developer docs style guide (contractions, future tense, wordiness)

![Apps](https://res.cloudinary.com/dpenc2lit/image/upload/v1687367319/mycology_fb_bpfmh3.png)

> Search for "Pipedream", and add the app to your Facebook Group. Once the Pipedream app has successfully been added to your group, you can start building automated workflows on Pipedream using Facebook Groups.

Reviewer: ```suggestion
Search for "Pipedream", and add the app to your Facebook Group. Once the Pipedream app has been successfully added to your group, you can start building automated workflows on Pipedream using Facebook Groups.
```

Reviewer: Changed 'has successfully been added' to 'has been successfully added' (Dreyer's English: clarity)

> Search for "Pipedream", and add the app to your Facebook Group. Once the Pipedream app has been successfully added to your group, you can start building automated workflows on Pipedream using Facebook Groups.

Reviewer: ```suggestion
Search for "Pipedream", and add the app to your Facebook Group. Once the Pipedream app is successfully added to your group, you can start building automated workflows on Pipedream using Facebook Groups.
```

Reviewer: Changed 'has been' to 'is' - Google's Developer docs style guide (present tense)
